src/meek.jl

```diff
@@ -0,0 +1,104 @@
+# http://proceedings.mlr.press/v89/katz19a/katz19a-supp.pdf
```

Comment:
In this files I have my implementation of the meek rules 1-3, I added rule number 4 but I am confused why it wasn't used in the PC algorithm (which I have from https://jmlr.org/papers/volume15/colombo14a/colombo14a.pdf)

Reply:
Rule 4 is not needed to obtain the CPDAG from skeleton + v-structures, so your implementation was correct. It is only needed in case of background knowledge, i.e. some edge orientations which do not follow from v-structures. 

So it was also not needed in the implementation here, were orientation also starts with skeleton + v-structures.